src/GafferSceneUIModule/SetEditorBinding.cpp

```diff
@@ -407,6 +407,422 @@ class SetNameColumn : public StandardPathColumn
 
 };
 
+//////////////////////////////////////////////////////////////////////////
+// VisibleSetInclusionsColumn - displays and modifies inclusions membership
+// of the VisibleSet in the provided context.
+//////////////////////////////////////////////////////////////////////////
+
+class VisibleSetInclusionsColumn : public PathColumn
+{
+
+	public :
+
+		IE_CORE_DECLAREMEMBERPTR( VisibleSetInclusionsColumn )
+
+		VisibleSetInclusionsColumn( ContextPtr context )
+			:	PathColumn(), m_context( context )
+		{
+			buttonPressSignal().connect( boost::bind( &VisibleSetInclusionsColumn::buttonPress, this, ::_3 ) );
+			buttonReleaseSignal().connect( boost::bind( &VisibleSetInclusionsColumn::buttonRelease, this, ::_1, ::_2, ::_3 ) );
+			m_context->changedSignal().connect( boost::bind( &VisibleSetInclusionsColumn::contextChanged, this, ::_2 ) );
+		}
+
+		CellData cellData( const Gaffer::Path &path, const IECore::Canceller *canceller ) const override
+		{
+			CellData result;
+
+			auto setPath = IECore::runTimeCast<const SetPath>( &path );
+			if( !setPath )
+			{
+				return result;
+			}
+
+			const auto setName = runTimeCast<const IECore::StringData>( setPath->property( g_setNamePropertyName ) );
+			if( !setName )
+			{
+				// We only interact with locations representing sets
+				return result;
+			}
+
+			auto iconData = new CompoundData;
+			iconData->writable()["state:highlighted"] = g_setIncludedHighlightedTransparentIconName;
+			result.icon = iconData;
+			result.toolTip = g_inclusionToolTip;
+
+			const auto visibleSet = ContextAlgo::getVisibleSet( m_context.get() );
+			if( visibleSet.inclusions.isEmpty() )
+			{
+				result.value = new IntData( 0 );
+				return result;
+			}
+
+			const auto setMembers = setPath->getScene()->set( setName->readable() );
```

Comment:
We need to scope the right context before calling `set()` here.

Reply:
Fixed in e5a5c76.